packages/amplify_core/android/build.gradle

```diff
@@ -4,6 +4,7 @@ version '1.0-SNAPSHOT'
 buildscript {
     ext.kotlin_version = '1.3.50'
     repositories {
+        mavenLocal()
```

Comment:
Please remove the references to mavenLocal.

Reply:
Removing these will prevent us from testing on Jameson's flutter ampilfy-android branch.  Can we just keep this step for the final merge?

Reply:
Ok - As long as we remember to remove them.